docs/en/advanced_guides/datasets/kitti_det.md

```diff
@@ -182,30 +183,30 @@ An example to test PointPillars on KITTI with 8 GPUs and generate a submission t
 
 - First, you need to modify the `test_evaluator` dict in your config file to add `pklfile_prefix` and `submission_prefix`, just like:
 
-```python
-data_root = 'data/kitti/'
-test_evaluator = dict(
-    type='KittiMetric',
-    ann_file=data_root + 'kitti_infos_test.pkl',
-    metric='bbox',
-    pklfile_prefix='results/kitti-3class/kitti_results',
-    submission_prefix='results/kitti-3class/kitti_results')
-```
+  ```python
+  data_root = 'data/kitti/'
```

Comment:
```python
data_root = 'data/kitti/'
test_dataloader = dict(
    dataset=dict(ann_file='kitti_infos_test.pkl', load_eval_anns=False, data_prefix=dict(img='testing/image_2')))
test_evaluator = dict(
    ann_file=data_root + 'kitti_infos_test.pkl',
    format_only=True,
    pklfile_prefix='results/kitti-mono3d/kitti_results',
    submission_prefix='results/kitti-mono3d/kitti_results'
    )
```

Reply:
Please also update the corresponding Chinese doc

Reply:
We updated the KITTI metric in https://github.com/open-mmlab/mmdetection3d/pull/2005